src/tritonserver.cc

```diff
@@ -922,6 +922,20 @@ TRITONSERVER_InferenceTraceActivityString(
   return "<unknown>";
 }
 
+TRITONAPI_DECLSPEC const char*
+TRITONSERVER_InferenceTraceModeString(
+    TRITONSERVER_InferenceTraceMode mode)
+{
+  switch (mode) {
+    case TRITONSERVER_TRACE_MODE_TRITON:
+      return "TRITON";
```

Comment:
This looks fine to me. Unless @GuanLuo you have a better suggestion for the original tracer mode.